packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.stories.tsx

```diff
@@ -41,20 +32,21 @@ const storyWithProps = (ctx: AppContextValue, history?: History) => {
   return (
     <AppContext.Provider value={ctx}>
       <LocationProvider {...{ history }}>
-        <AccountRecoveryResetPassword />
+        <AccountRecoveryResetPassword
+          integration={createMockAccountRecoveryResetPasswordSyncDesktopIntegration()}
```

Comment:
Might have mentioned this in passing. I think this name could be shortened, since the mock is internal to this page folder.

Reply:
I think this was the exact one I messaged you about because it became absurdly long, lol. Since I followed a pattern already set I think for now I'm going to leave it, we can adjust during the reset PW redesign or in a follow up ticket. I'm curious what you think about dropping the name `create` or if that feels like a keyword for you over just referencing `mock`. I'm fine with this becoming `createMockSyncDesktopIntegration` or `mockSyncDesktopIntegration` and then yeah, if we need to search the codebase we can just look at directory names to see where it corresponds to.